templates/rackspace-identity-basic-auth.cfg.xml.erb

```diff
@@ -5,6 +5,6 @@
     rackspace-identity-service-uri="<%= @identity_service_url %>"
     token-cache-timeout-millis="<%= @token_cache_timeout %>">
 <%- if @delegating -%>
-    <delegating/>
+    <delegating<%- if ! @delgating_quality.nil? -%>  quality="<%= @delegating_quality %>"<%- end -%>/>
```

Comment:
Careful with the -%> as it'll nuke that space before quality.  In this case, it's best not to put the - here